lib/perl/Genome/Model/Tools/Manta/Base.pm

```diff
@@ -0,0 +1,246 @@
+package Genome::Model::Tools::Manta::Base;
+
+use strict;
+use warnings;
+
+use Genome;
+use Carp qw(confess);
+
+class Genome::Model::Tools::Manta::Base {
+    is => 'Command',
+    is_abstract => 1,
+    attributes_have => [
+        tool_arg_name => {
+            is => 'String',
+            is_optional => 1,
+        },
+        tool_input_file => {
+            is => 'Boolean',
+            is_optional => 1,
+        },
+    ],
+    has_param => [
+        version => {
+            is => 'Version',
+            doc => 'The version of Manta to use.',
+            valid_values => ['0.29.6'],
+        },
+    ],
+    has_optional_output => [
+        output_files => {
+            is_many => 1,
+        },
+        input_files => {
+            is_many => 1,
+        },
+    ],
+};
+
+sub help_detail {
+    "Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads."
+}
+
+my %TOOL_VERSIONS = (
+    '0.29.6'     => '/gscmnt/gc13001/info/model_data/jwalker_scratch/src/manta-0.29.6.centos5_x86_64/bin',
```

Comment:
Shown below is an example changes I made to demonstrate the use of the manta docker image.

diff --git a/lib/perl/Genome/Model/Tools/Manta/Base.pm b/lib/perl/Genome/Model/Tools/Manta/Base.pm
index 534ace1..cb98198 100644
--- a/lib/perl/Genome/Model/Tools/Manta/Base.pm
+++ b/lib/perl/Genome/Model/Tools/Manta/Base.pm
@@ -7,7 +7,7 @@ use Genome;
 use Carp qw(confess);
 
 my %TOOL_VERSIONS = (
-    '0.29.6'     => '/gscmnt/gc13001/info/model_data/jwalker_scratch/src/manta-0.29.6.centos5_x86_64/bin',
+    '0.29.6'     => 'bsub -Is -q docker -a "docker(registry.gsc.wustl.edu/sleong/manta:0.29.6)" /usr/local/pkg/manta-0.29.6.centos5_x86_64/bin/',
 );
 
 class Genome::Model::Tools::Manta::Base {
@@ -56,7 +56,7 @@ sub path_for_version {
 
 sub tool_path {
     my $self = shift;
-    return File::Spec->join($self->path_for_version($self->version),$self->_tool_subcommand_name);
+    return $self->path_for_version($self->version) . $self->_tool_subcommand_name;
 }
 
 # tool subcommand name as a string (e.g., 'configManta.py')